### ISSUE TITLE
Redirect after authentication on http://localhost without /

### ISSUE THREAD

**Issue by atzimot** [writer/writer-framework#501]
After a succesful auth the app doesn't redirect to the base url
I redirecting to /authorize?code= and is displaying a white page
If I delete  /authorize?code= the app loads
What do i have to do ?

oidc = writer.auth.Oidc(
    client_id="xxx",
    client_secret="xxxx",
    host_url=os.getenv('HOST_URL', "http://localhost:33081"),
    url_authorize="http://localhost:33050/connect/authorize",
    url_oauthtoken="http://localhost:33050/connect/token",
    url_userinfo='http://localhost:33050/connect/userinfo',
    scope="openid profile common"
)

This is my settings

**Comment by FabienArcellier** [comment]
At the end of the authentication flow, your browser receives a `307 Temporary Redirect` redirect to `http://localhost:33081`. Is this response visible in your browser's network inspector? could you share a screenshot of the network inspector ?

**Comment by atzimot** [comment]
![image](https://github.com/user-attachments/assets/9a21bb1d-82a9-4fae-bd13-2863d1c6a8fa)

![image](https://github.com/user-attachments/assets/274c58ae-6247-42b2-b4ea-ee9330814c26)

**Comment by FabienArcellier** [comment]
As the location looks empty, the I suppose HOST_URL environment variable exists and contains an empty string "".

```python
host_url = os.getenv('HOST_URL', "http://localhost:33081")
if host_url is not None and host_url.strip() == "":
    host_url="http://localhost:33081"

oidc = writer.auth.Oidc(
    client_id="xxx",
    client_secret="xxxx",
    host_url=host_url,
    url_authorize="http://localhost:33050/connect/authorize",
    url_oauthtoken="http://localhost:33050/connect/token",
    url_userinfo='http://localhost:33050/connect/userinfo',
    scope="openid profile common"
)
```

**Comment by atzimot** [comment]
no smae issue the app doesn't redirect to http://localhost:33081/. If i delete the part after / the app loads

**Comment by atzimot** [comment]
I have found that in auth.py on line 213 if host_url_path='' then the redirect doesn't work but if host_url_path='/' works

**Comment by FabienArcellier** [comment]
I will investigate what we can do to make this situation more easy to handle.

**Comment by FabienArcellier** [comment]
the fix has been released in `0.7.0rc2`

```bash
pip install writer==0.7.0rc2
```

**Comment by atzimot** [comment]
thanks I will test
